### ISSUE TITLE
Reader New Post Notifications: link to site stream rather than author site?

### ISSUE THREAD

**Issue by bluefuton** [Automattic/wp-calypso#21718]
I tried subscribing to new post notifications for Time.com, and then promptly forgot about it.

The next day, I woke up to an avalanche of notifications from a variety of different authors, like this:

<img width="435" alt="screen shot 2018-01-22 at 12 56 51" src="https://user-images.githubusercontent.com/17325/35209506-5218b5aa-ff74-11e7-8414-7a35e747ceaa.png">

It wasn't clear which site they came from, and how I could easily unsubscribe. I later realised that I could click the post title instead to access notification settings, but that didn't seem intuitive.

Should we link the author name to the Reader rather than to the author's own site?

**Comment by jancavan** [comment]
> It wasn't clear which site they came from, and how I could easily unsubscribe. I later realised that I could click the post title instead to access notification settings, but that didn't seem intuitive.

Agreed. As mentioned before, users may also try to change these settings by clicking on the cog icon. This jumps to `/me/notifications` where post notification settings could not be found. This is definitely something we'll be addressing.

> Should we link the author name to the Reader rather than to the author's own site?

I like this idea. If they don't have a site, it just links to time.com. This should also just jump to time.com's Reader site stream.

**Comment by jancavan** [comment]
> It wasn't clear which site they came from, and how I could easily unsubscribe.

This is also especially confusing if you're logged-in, but on the front-end.

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

As there has been no recent activity on this issue it is being closed with the label `Closed - Won't Fix`.
If this issue should remain open, please feel free to re-open this bug.
